connectivity/nanostack/mbed-mesh-api/source/NanostackPPPInterface.cpp

```diff
@@ -17,7 +17,7 @@
 
 #include "NanostackPPPInterface.h"
 #include "NanostackPPPPhy.h"
-#include "PPP.h"
```

Comment:
The class we want to include is located at `netsocket/PPP.h`. `ppp/ppp.h` is a very different file.

Reply:
I changed and added the pre - sub folder .. but still it didn't resolve the issue. 
There are several things that might work on window but not on linux and vice versa.
I complained about the export from the online IDE. Believe me that I cannot compile any project without using a combination of Windows (eclipse) and linux in the same time for compiling just one project.. some files compiles using windows and get problem later . and I continue compiling in linux and fails there also .. return to windows ..etc.
So, what is the cause of these problems? might be many things. Bad names, wrong searching path or includes, ..etc.
thanks.
command prompts crashes during the link time of the .elf file. always.